src/components/program/program-form/ProgramForm.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { useFormik } from "formik";
+
+// import validationSchema from "./validation";
+import { Form, Field, Button, Pane } from "../../ui";
+
+const ProgramForm = ({ initialValues, onSubmit, isLoading }) => {
+  const {
+    handleSubmit,
+    handleBlur,
+    handleChange,
+    touched,
+    values,
+    errors,
+  } = useFormik({
+    initialValues: {
+      title: initialValues.title || "",
+      description: initialValues.description || "",
+      days: initialValues.days || 1,
+      isPrivate: initialValues.isPrivate || false,
+    },
+    // validationSchema: validationSchema,
+    onSubmit: (values) => onSubmit(values),
```

Comment:
Be careful in that case because we should be able to use this form for a creation but also for an updating. So IMO it should be the parent to take care of the action